### PR TITLE
Feature/#17 initialise data for tests

### PR DESCRIPTION
**Pull request by HongShi10**
## Pull Request Information
**Relates to Github Issue:** #17 

**What is the PR Type:** 
- feature

**Describe the added/fixed behaviour:**
Added dummy data to be loaded into the database so that the implemented unit tests can use the dummy data.

## Checklist

- [x] Pull request has suitable title, adequately described and associated with a github issue
- [x] Branch name follows correct convention (e.g. feature/[id]-some-feature)
- [x] Branch builds and executes with no errors
- [ ] Test cases has been generated for the feature and test cases pass
- [ ] Updated relevant wiki/documentation

## Additional Information
Wiki documentation will needed to define the new many to many relationships and test cases once implemented 

## Final Reminders
Please remember to squash the commits

### CODE REVIEW

**Comment by HongShi10** [comment]
Also I was wondering if there was a better way to do the initialisation of data

File: WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs [hunk 1]
```diff
@@ -8,19 +8,161 @@ namespace WebApiBackend.Helpers
...
+                Fixed = true,
+                Frequency = Frequency.Monthly,
+                StartDate = new DateTime(2020, 03, 01),
+                EndDate = new DateTime(2020, 10, 01),
+            };
+            userPaymentBryan1 = new UserPayment
+            {
+                Payment = payment1,
+                User = user1,
+                UserName = user1.UserName,
```
Comment (mitchfaulcon) [inline WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs]: Should this be user3 instead of user1?

Comment (HongShi10) [reply WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs]: 😮 Yes it should, ill change that

Comment (mitchfaulcon) [reply WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs]: You changed the wrong one hahah, switch lines 82-83 with 97-98

Comment (HongShi10) [reply WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs]: Oh i thought I changed Bryan, I’ll do that asap

**Comment by HongShi10** [comment]
I assumed that both functions would be called in the test initialization or should I just add it into the load database function

File: WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs (hunk 1 above)
Comment (HongShi10) [reply WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs]: @mitchfaulcon I changed it, can you double check that I havent missed out on anything else

**Pull request merged**